.pre-commit-hooks.yaml

```diff
@@ -10,7 +10,7 @@
         Prevent common mistakes of `assert mck.not_called()`, `assert mck.called_once_with(...)` and
         and `mck.assert_called`.
     language: pygrep
-    entry: '(assert .*\.(not_called|called\w*)|\.assert_(not_called|called\w*)($|[^(\w]))'
+    entry: '(assert .*\.(not_called|called_)|\.assert_(not_called|called|called_\w+)($|[^(\w]))'
```

Comment:
instead of being cute with regex let's be explicit. here's the list of things to `|`:

```
>>> [k for k in dir(mock.Mock()) if k.startswith('assert_')]
['assert_any_call', 'assert_called', 'assert_called_once', 'assert_called_once_with', 'assert_called_with', 'assert_has_calls', 'assert_not_called']
```